### ISSUE TITLE
Disctrict renovation

### ISSUE THREAD

**Issue by Kastuk** [loiste-interactive/Obenseuer-Issues#1001]
Will be nice to renovate some of district elements around our tenement, like add fence to deadly canal, or add/fix streetlights, or clean up street camping when people got their appartments in the end.

**Comment by Orphiex** [comment]
I dunno, this seems like it would be thematically hard to implement within the current game? I guess the mechanism already exists, given that players can renovate the tenement's exterior environments, and it would be a cool element to incorporate into the game. And the "cleaning up street tents" part makes sense, although it would probably be a low priority to implement compared to the current issues with apartment renters.

However, from a narrative perspective, the player character has neither the authority to make renovations to the wider district, nor the motivation to finance them. It would have to be late-game content anyway, given that it shouldn't be accessible until the player meets a representative of either the Stalburg government or the Obenseuer shadow government. Plus, the main reason wealthy people fund these sorts of things IRL is to get tax breaks, which the player doesn't currently pay.

And from a gameplay perspective, while some of these district renovations might be helpful to prevent accidental player deaths, I personally haven't fallen into a canal even once. For me, funding these city improvements instead of paying for tenement renovations would have to come with some sort of in-game benefit to be useful as anything more than cosmetics, which would probably require the devs to implement a whole new mechanic. (Maybe access to a special VVIP store in the Bazaar if you "donate" enough to the Obenseuer Improvement Fund?)

Basically, while I think this would be cool, I imagine it'll be low-priority compared to implementing bugfixes and main-story content.

**Comment by Kastuk** [comment]
I think theres may be sort of communal projects when you got several handyman friends (like that bridge builder) and going to improve overal living of community.
Maybe quest line from Redemption Militia?

**Comment by greensalt99** [comment]
Different people like different stuff, so  Every Character will have different motivation. i think that cleaning/reshaping Obenseuer into more respectable shape would be neat think/(end goal?).  only question what happens with that new space and all those spawn points?
